packages/server/graphql/types/Organization.ts

```diff
@@ -166,7 +174,7 @@ const Organization: GraphQLObjectType<any, GQLContext> = new GraphQLObjectType<a
       resolve: async ({id: orgId}: {id: string}, _args: unknown, {dataLoader}: GQLContext) => {
         const organizationUsers = await dataLoader.get('organizationUsersByOrgId').load(orgId)
         const inactiveUserCount = organizationUsers.filter(
-          ({inactive}: OrganizationUser) => inactive
+          ({inactive}: TOrganizationUser) => inactive
```

Comment:
+1 same here - we don't need to add the type as it's already known (I realise you didn't add the type in the first place!). Same thing on line 192 below